stdlib/public/core/VarArgs.swift

```diff
@@ -290,7 +290,8 @@ final public class VaListBuilder {
     // supported vararg type is greater than the alignment of Int.
     // FIXME: this implementation is not portable because
     // alignof differs from the ABI alignment on some architectures
-#if os(watchOS) && arch(arm)   // FIXME: rdar://21203036 should be arch(armv7k)
+    // All ARM platforms with the exception of iOS use the AAPCS conventions
+#if !os(iOS) && arch(arm)   // FIXME: rdar://21203036 should be arch(armv7k)
```

Comment:
@jrose-apple git hub again lost all the comments in the update, but here's the corrected line.  I left the FIXME because that seemed to be your (slight?) preference.